packages/clarity-devtools/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "clarity-devtools",
-  "version": "0.7.8",
```

Comment:
there are 2 more places where we have the version - version.js and the manifest.json can you please change all this?

Reply:
Thanks Swaathee, done